show-land/sass/layout/_top-nav.scss

```diff
@@ -0,0 +1,30 @@
+.container {
+  .header {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .logo {
+    padding-top: 0.8rem;
+    font-size: 2rem;
+    font-weight: 700;
+  }
+
+  .main-nav {
+    &__list {
+      display: flex;
+      padding-top: 1rem;
+      gap: 1.5rem;
+      list-style: none;
+      align-items: center;
+    }
+    &__link:link,
+    &__link:visited {
+      text-decoration: none;
+      font-size: 1rem;
+      color: $colorwhite;
+      text-transform: uppercase;
+      font-weight: 700;
+    }
```

Comment:
Please again here you could use variables for gap, padding and font-weight and also font size